32bits/minimum-config.txt
# Mi configuración
set pan0=1528
calibrate
set pan0_calibrated=1
feature easing
set easing_steps=50
set init_servos=0
feature display
set p = 3200
set i = 50
set d = 200
set max_pid_error = 1
set max_pid_accumulator=5000
set max_pid_gain=500
set gps_provider = NMEA
set gps_sbas_mode = AUTO
set gps_auto_config = ON
set gps_auto_baud = OFF
set gps_min_sats=6
feature gps
feature -vbat
set telemetry_protocol=8
set mag_declination=207

 47° 24.215'N
  8° 32.151'E

# Configuración de Simba
# set
Current settings:
gps_baud = 1541
gps_provider = NMEA
gps_sbas_mode = AUTO
gps_auto_config = ON
gps_auto_baud = OFF
gps_min_sats = UBLOX
p = 2000
i = 15
d = 200
max_pid_error = 1
max_pid_accumulator = 50000
max_pid_gain = 500
pan0 = 1500
pan0_calibrated = 1
min_pan_speed = 0
offset = 270.000
init_servos = 0
tilt0 = 2025
tilt90 = 1050
easing = 2
easing_steps = 40
easing_min_angle = 4
easing_milis = 15
telemetry_baud = 1
telemetry_protocol = 8
start_tracking_distance = 10
mag_declination = 0 


ÚLTIMOS CAMBIOS:
(1.7.2)
Corregido 
Display encendido por defecto.
El Ticker se activa sólo cuando llegan datos por el puerto serie, y no estamos en modo servotest(cli)(1.7.2)
(1.7.3)
Botón calibración no funciona
Datos de posición negativos no salen bien. Ejemplo 0.-20150
(1.8.4)
GOS Local activa HOME implementado
(1.9.2)
Mostrar en el display el nombre de la telemetría activada.
Eliminado parpadeo del siplay si no tenemos vbat ni gps.
Declinación magnética no se está usando.

PARA CORREGIR

Problemas con pan0 al activar/desactivar init_servos (no sabemos si es realmente un bug, más bien parece un problema de configuración)

MEJORAS
sacar por un puerto softserial la telemetría recibida en protocolo mavlink, por ejemplo, y así poder usar aplicaciones tipo android/ios que nos permitan ver en el smartphone o en el iPad por donde está pululando nuestro areomodelo

32bits/clean-flight-config.txt
dump

# version
# Cleanflight/NAZE 1.12.0 Jan 11 2016 / 16:02:07 (7a4bc02)
# dump master

# mixer
mixer QUADX
mmix reset
smix reset


# feature
feature -RX_PPM
feature -VBAT
feature -INFLIGHT_ACC_CAL
feature -RX_SERIAL
feature -MOTOR_STOP
feature -SERVO_TILT
feature -SOFTSERIAL
feature -GPS
feature -FAILSAFE
feature -SONAR
feature -TELEMETRY
feature -CURRENT_METER
feature -3D
feature -RX_PARALLEL_PWM
feature -RX_MSP
feature -RSSI_ADC
feature -LED_STRIP
feature -DISPLAY
feature -ONESHOT125
feature -BLACKBOX
feature -CHANNEL_FORWARDING
feature VBAT
feature GPS
feature FAILSAFE
feature RX_PARALLEL_PWM
feature DISPLAY


# map
map AETR1234


# serial
serial 0 1 115200 57600 0 115200
serial 1 2 115200 38400 0 115200


# led
led 0 15,15:ES:IA:0
led 1 15,8:E:WF:0
led 2 15,7:E:WF:0
led 3 15,0:NE:IA:0
led 4 8,0:N:F:0
led 5 7,0:N:F:0
led 6 0,0:NW:IA:0
led 7 0,7:W:WF:0
led 8 0,8:W:WF:0
led 9 0,15:SW:IA:0
led 10 7,15:S:WF:0
led 11 8,15:S:WF:0
led 12 7,7:U:WF:0
led 13 8,7:U:WF:0
led 14 7,8:D:WF:0
led 15 8,8:D:WF:0
led 16 8,9::R:3
led 17 9,10::R:3
led 18 10,11::R:3
led 19 10,12::R:3
led 20 9,13::R:3
led 21 8,14::R:3
led 22 7,14::R:3
led 23 6,13::R:3
led 24 5,12::R:3
led 25 5,11::R:3
led 26 6,10::R:3
led 27 7,9::R:3
led 28 0,0:::0
led 29 0,0:::0
led 30 0,0:::0
led 31 0,0:::0


# color
color 0 0,0,0
color 1 0,255,255
color 2 0,0,255
color 3 30,0,255
color 4 60,0,255
color 5 90,0,255
color 6 120,0,255
color 7 150,0,255
color 8 180,0,255
color 9 210,0,255
color 10 240,0,255
color 11 270,0,255
color 12 300,0,255
color 13 330,0,255
color 14 0,0,0
color 15 0,0,0

set looptime = 3500
set emf_avoidance = OFF
set i2c_overclock = OFF
set gyro_sync = OFF
set gyro_sync_denom = 1
set mid_rc = 1500
set min_check = 1100
set max_check = 1900
set rssi_channel = 0
set rssi_scale = 30
set rssi_ppm_invert = OFF
set rc_smoothing = ON
set input_filtering_mode = OFF
set min_throttle = 1150
set max_throttle = 1850
set min_command = 1000
set servo_center_pulse = 1500
set 3d_deadband_low = 0
set 3d_deadband_high = 0
set 3d_neutral = 0
set 3d_deadband_throttle = 0
set motor_pwm_rate = 400
set servo_pwm_rate = 50
set retarded_arm = OFF
set disarm_kill_switch = ON
set auto_disarm_delay = 5
set small_angle = 25
set fixedwing_althold_dir = 1
set reboot_character = 82
set gps_provider = NMEA
set gps_sbas_mode = AUTO
set gps_auto_config = ON
set gps_auto_baud = OFF
set serialrx_provider = SPEK1024
set sbus_inversion = ON
set spektrum_sat_bind = 0
set telemetry_switch = OFF
set telemetry_inversion = OFF
set frsky_default_lattitude =  0.000
set frsky_default_longitude =  0.000
set frsky_coordinates_format = 0
set frsky_unit = IMPERIAL
set frsky_vfas_precision = 0
set hott_alarm_sound_interval = 5
set battery_capacity = 0
set vbat_scale = 110
set vbat_max_cell_voltage = 43
set vbat_min_cell_voltage = 33
set vbat_warning_cell_voltage = 35
set current_meter_scale = 400
set current_meter_offset = 0
set multiwii_current_meter_output = OFF
set current_meter_type = ADC
set align_gyro = DEFAULT
set align_acc = DEFAULT
set align_mag = DEFAULT
set align_board_roll = 0
set align_board_pitch = 0
set align_board_yaw = 0
set max_angle_inclination = 500
set gyro_lpf = 42HZ
set moron_threshold = 32
set imu_dcm_kp = 2500
set imu_dcm_ki = 0
set yaw_control_direction = 1
set pid_at_min_throttle = ON
set yaw_motor_direction = 1
set yaw_jump_prevention_limit = 200
set tri_unarmed_servo = ON
set servo_lowpass_freq = 400
set servo_lowpass_enable = OFF
set failsafe_delay = 10
set failsafe_off_delay = 200
set failsafe_throttle = 1000
set failsafe_kill_switch = OFF
set failsafe_throttle_low_delay = 100
set failsafe_procedure = 0
set rx_min_usec = 885
set rx_max_usec = 2115
set acc_hardware = 0
set baro_hardware = 0
set mag_hardware = 0
set blackbox_rate_num = 1
set blackbox_rate_denom = 1
set blackbox_device = SERIAL
set magzero_x = 0
set magzero_y = 0
set magzero_z = 0

# rxfail
rxfail 0 a
rxfail 1 a
rxfail 2 a
rxfail 3 a
rxfail 4 h
rxfail 5 h
rxfail 6 h
rxfail 7 h
rxfail 8 h
rxfail 9 h
rxfail 10 h
rxfail 11 h
rxfail 12 h
rxfail 13 h
rxfail 14 h
rxfail 15 h
rxfail 16 h
rxfail 17 h

# dump profile

# profile
profile 0

# aux
aux 0 0 0 900 900
aux 1 0 0 900 900
aux 2 0 0 900 900
aux 3 0 0 900 900
aux 4 0 0 900 900
aux 5 0 0 900 900
aux 6 0 0 900 900
aux 7 0 0 900 900
aux 8 0 0 900 900
aux 9 0 0 900 900
aux 10 0 0 900 900
aux 11 0 0 900 900
aux 12 0 0 900 900
aux 13 0 0 900 900
aux 14 0 0 900 900
aux 15 0 0 900 900
aux 16 0 0 900 900
aux 17 0 0 900 900
aux 18 0 0 900 900
aux 19 0 0 900 900

# adjrange
adjrange 0 0 0 900 900 0 0
adjrange 1 0 0 900 900 0 0
adjrange 2 0 0 900 900 0 0
adjrange 3 0 0 900 900 0 0
adjrange 4 0 0 900 900 0 0
adjrange 5 0 0 900 900 0 0
adjrange 6 0 0 900 900 0 0
adjrange 7 0 0 900 900 0 0
adjrange 8 0 0 900 900 0 0
adjrange 9 0 0 900 900 0 0
adjrange 10 0 0 900 900 0 0
adjrange 11 0 0 900 900 0 0

# rxrange
rxrange 0 1000 2000
rxrange 1 1000 2000
rxrange 2 1000 2000
rxrange 3 1000 2000

# servo
servo 0 1000 2000 1500 45 45 100 -1
servo 1 1000 2000 1500 45 45 100 -1
servo 2 1000 2000 1500 45 45 100 -1
servo 3 1000 2000 1500 45 45 100 -1
servo 4 1000 2000 1500 45 45 100 -1
servo 5 1000 2000 1500 45 45 100 -1
servo 6 1000 2000 1500 45 45 100 -1
servo 7 1000 2000 1500 45 45 100 -1

set gps_pos_p = 15
set gps_pos_i = 0
set gps_pos_d = 0
set gps_posr_p = 34
set gps_posr_i = 14
set gps_posr_d = 53
set gps_nav_p = 25
set gps_nav_i = 33
set gps_nav_d = 83
set gps_wp_radius = 200
set nav_controls_heading = ON
set nav_speed_min = 100
set nav_speed_max = 300
set nav_slew_rate = 30
set alt_hold_deadband = 40
set alt_hold_fast_change = ON
set deadband = 0
set yaw_deadband = 0
set throttle_correction_value = 0
set throttle_correction_angle = 800
set default_rate_profile = 0
set gimbal_mode = NORMAL
set acc_cut_hz = 15
set accxy_deadband = 40
set accz_deadband = 40
set accz_lpf_cutoff =  5.000
set acc_unarmedcal = ON
set acc_trim_pitch = 0
set acc_trim_roll = 0
set baro_tab_size = 21
set baro_noise_lpf =  0.600
set baro_cf_vel =  0.985
set baro_cf_alt =  0.965
set mag_declination = 0
set pid_controller = MWREWRITE
set p_pitch = 40
set i_pitch = 30
set d_pitch = 23
set p_roll = 40
set i_roll = 30
set d_roll = 23
set p_yaw = 85
set i_yaw = 45
set d_yaw = 0
set p_pitchf =  1.400
set i_pitchf =  0.400
set d_pitchf =  0.030
set p_rollf =  1.400
set i_rollf =  0.400
set d_rollf =  0.030
set p_yawf =  3.500
set i_yawf =  0.400
set d_yawf =  0.010
set level_horizon =  3.000
set level_angle =  5.000
set sensitivity_horizon = 75
set p_alt = 50
set i_alt = 0
set d_alt = 0
set p_level = 90
set i_level = 10
set d_level = 100
set p_vel = 120
set i_vel = 45
set d_vel = 1
set yaw_p_limit = 500
set gyro_soft_lpf = OFF
set dterm_cut_hz = 0
set yaw_pterm_cut_hz = 0
set gtune_loP_rll = 10
set gtune_loP_ptch = 10
set gtune_loP_yw = 10
set gtune_hiP_rll = 100
set gtune_hiP_ptch = 100
set gtune_hiP_yw = 100
set gtune_pwr = 0
set gtune_settle_time = 450
set gtune_average_cycles = 16

# dump rates

# rateprofile
rateprofile 0

set rc_rate = 90
set rc_expo = 65
set rc_yaw_expo = 0
set thr_mid = 50
set thr_expo = 0
set roll_rate = 0
set pitch_rate = 0
set yaw_rate = 0
set tpa_rate = 0
set tpa_breakpoint = 1500

# ##dump

amv-open360tracker/settings.cpp
#include "inttypes.h"
#include <EEPROM.h>
#include "settings.h"
#include "defines.h"
#include <Arduino.h>

uint8_t EEPROM_DEFAULT[EEPROM_SETTINGS]={
/* 00 */ FMW_VER,
/* 01 */ DEF_S_PID_P,
/* 02 */ DEF_S_PID_I,
/* 03 */ DEF_S_PID_D,
/* 04 */ DEF_S_MAX_PID_ERROR,
/* 05 */ DEF_S_TILT_0,
/* 06 */ DEF_S_TILT_90,
/* 07 */ DEF_S_TILT_EASING,
/* 08 */ DEF_S_TILT_EASING_STEPS, 
/* 09 */ DEF_S_TILT_EASING_MIN_ANGLE, 
/* 10 */ DEF_S_TILT_EASING_MILIS, 
/* 11 */ DEF_S_PAN_0,
/* 12 */ DEF_S_MIN_PAN_SPEED,
/* 13 */ DEF_S_DECLINATION,
/* 14 */ DEF_S_OFFSET,
/* 15 */ DEF_S_LOCAL_GPS,
/* 16 */ DEF_S_GPS_MODEL,
/* 17 */ DEF_S_GPS_BAUDRATE,
/* 18 */ DEF_S_START_TRACKING_DISTANCE,
/* 19 */ DEF_S_LCD_DISPLAY,
/* 20 */ DEF_S_LCD_SIZE_ROW,
/* 21 */ DEF_S_LCD_I2C_ADDR,
/* 22 */ DEF_S_LCD_MODEL,
/* 23 */ DEF_S_BATTERYMONITORING,
/* 24 */ DEF_S_BATTERYMONITORING_RESISTOR_1,
/* 25 */ DEF_S_BATTERYMONITORING_RESISTOR_2,
/* 26 */ DEF_S_BATTERYMONITORING_CORRECTION,
/* 27 */ DEF_S_SERVOTEST,
/* 28 */ DEF_S_CLI
};

uint8_t Settings[EEPROM_SETTINGS];
char *param_names[EEPROM_SETTINGS]={
/* 00 */"check",
/* 01 */"P",
/* 02 */"I",
/* 03 */"D",
/* 04 */"max_pid_error",
/* 05 */"tilt0",
/* 06 */"tilt90",
/* 07 */"easing",
/* 08 */"easing_steps",
/* 09 */"easing_min_angle",
/* 10 */"easing_milis",
/* 11 */"pan0",
/* 12 */"min_pan_speed",
/* 13 */"declination",
/* 14 */"offset",
/* 15 */"gps",
/* 16 */"gps_model",
/* 17 */"gps_bauds",
/* 18 */"start_track_dist",
/* 19 */"lcd",
/* 20 */"lcd_rows",
/* 21 */"lcd_addr",
/* 22 */"lcd_model",
/* 23 */"bat",
/* 24 */"bat_res1",
/* 25 */"bat_res2",
/* 26 */"bat_corr",
/* 27 */"servotest",
/* 28 */"cli"
 };
void writeEEPROM(void)
{
  for(uint8_t en=0;en<EEPROM_SETTINGS;en++){
    EEPROM.write(en+ADDR_OFFSET,Settings[en]);
  } 
  EEPROM.write(0+ADDR_OFFSET,FMW_VER);
}

uint8_t readEEPROM(void)
{
  for(uint8_t en=0;en<EEPROM_SETTINGS;en++){
     Settings[en] = EEPROM.read(en+ADDR_OFFSET);
  }
  return 1;
}
uint8_t checkEEPROM(void)
{
  uint8_t EEPROM_Loaded = EEPROM.read(0+ADDR_OFFSET);
  if (EEPROM_Loaded!=FMW_VER){
    defaultsEEMPROM();
  }
  return 1;
}
void defaultsEEMPROM(void)
{
  for(uint8_t en=0;en<EEPROM_SETTINGS;en++){
      EEPROM.write(en+ADDR_OFFSET,EEPROM_DEFAULT[en]);
  }
  readEEPROM();
}
void dumpSettings(){
  int param_value;
  Serial.println(F("List settings: "));
  for(uint8_t i=1;i<EEPROM_SETTINGS;i++){
    param_value=getParamValue(param_names[i]);
    Serial.print("set ");Serial.print(param_names[i]);Serial.print("=");Serial.println(param_value);
    Serial.flush();
  }
}
uint8_t setParamValue(String param_name,int param_value){
  uint8_t index = getParamIndex(param_name);
  int value=0;
  uint8_t divider;

  divider=(index==S_PID_P || index==S_PID_I || index==S_PID_D || index==S_TILT_0 || index==S_TILT_90 || index==S_PAN_0)?10:1;
  divider=(index==S_BATTERYMONITORING_RESISTOR_1 || index==S_BATTERYMONITORING_RESISTOR_2 || index==S_GPS_BAUDRATE || index==S_OFFSET)?100:divider;

  value=(index==S_MIN_PAN_SPEED && Settings[index]>100)?abs(param_value)+100:param_value/divider;
  value=(index==S_BATTERYMONITORING_CORRECTION)?param_value*10:value;
  
  Settings[index]=value;
}
int getParamValue(String param_name){
  uint8_t index = getParamIndex(param_name);
  int value=0;
  uint8_t multiplier;
  
  multiplier=(index==S_PID_P || index==S_PID_I || index==S_PID_D || index==S_TILT_0 || index==S_TILT_90 || index==S_PAN_0)?10:1;
  multiplier=(index==S_BATTERYMONITORING_RESISTOR_1 || index==S_BATTERYMONITORING_RESISTOR_2 || index==S_GPS_BAUDRATE || index==S_OFFSET)?100:multiplier;
    
  value=(index==S_MIN_PAN_SPEED && Settings[index]>100)?100-Settings[index]:Settings[index]*multiplier;
  value=(index==S_BATTERYMONITORING_CORRECTION)?(int)((float)Settings[index]/10):value;

  return value;
}
uint8_t getParamIndex(String param_name){
  uint8_t found = false;
  for(uint8_t i=1;i<EEPROM_SETTINGS;i++){
    for(uint8_t x=0;x<param_name.length();x++)
    {
      if(param_names[i][x]==param_name[x])
        found=true;
      else {
        found=false;
        break;
      }
    }
    if(found==true)
      return i;
  }
  Serial.println(F("Parameter not found"));
  return 0;
}

